script.js

```diff
@@ -0,0 +1,13 @@
+const buttons = document.getElementById("buttonsBlock");
+const result = document.getElementById("result");
+
+buttonClick = (event) => {
```

Comment:
Wow, you made the function without declaration (const or let). That's bad practice!

Reply:
I forgot about this. Sorry